runtime/gpufortrt/src/gpufortrt_api_core.f90

```diff
@@ -292,4 +292,31 @@ function gpufortrt_use_device_c_impl(hostptr,condition,if_present) &
     resultptr = gpufortrt_use_device_c_impl(hostptr,opt_if_arg,opt_if_present_arg)
   end function
 
+  function gpufortrt_is_present(data_arg, bytes) result(isOnDeviceMemory) 
+    use iso_c_binding
+    !
+    type(*), target, dimension(..)::data_arg
+    integer(c_int),value,intent(in) :: bytes
+    !
+    logical::isOnDeviceMemory
+    !
+    interface
+      function gpufortrt_is_present_c_impl(data_arg, bytes) &
+        bind(c,name="gpufortrt_present") result(isOnDeviceMemory) 
+          use iso_c_binding
```

Comment:
Also use `implicit none` here.

Reply:
Could we use implicit none once at the module scope? This way we don't need to repeat it for every interface/function/subroutine

Reply:
fixed in commit https://github.com/ROCmSoftwarePlatform/gpufort/pull/45/commits/59d4293c415a47f93867135dddb827fb1fd7a4ae

Reply:
You could do that (works both for `program` and `module`) but it would not apply to the interfaces; see
https://stackoverflow.com/a/24337734
Which can cause confusion.

Reply:
Ok then, I avoid putting it at module scope to keep it clear